projects/igniteui-angular/src/lib/grids/group-by-area/group-by-area.component.ts

```diff
@@ -0,0 +1,160 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    HostBinding,
+    Input,
+    Pipe,
+    PipeTransform,
+    QueryList,
+    TemplateRef,
+    ViewChildren
+} from '@angular/core';
+import { IChipsAreaReorderEventArgs, IgxChipComponent } from '../../chips/public_api';
+import { DisplayDensity } from '../../core/displayDensity';
+import { PlatformUtil } from '../../core/utils';
+import { IGroupingExpression } from '../../data-operations/grouping-expression.interface';
+import { ISortingExpression } from '../../data-operations/sorting-expression.interface';
+import { FlatGridType } from '../common/grid.interface';
+
+/**
+ * An internal component representing the group-by drop area for the igx-grid component.
+ *
+ * @hidden @internal
+ */
+@Component({
+    changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
Reordering chips in the group by area doesn't work very well with OnPush strategy. That's why we have removed it in the Tree Grid Group By Area [PR](https://github.com/IgniteUI/igniteui-angular/pull/9625/files?file-filters%5B%5D=.ts#diff-b8e9720ade38180ddb6381d29f61a589f72ee69f7a606e120a44bd459891d10c).